proj/src/main/webapp/static/oldchat.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
I'm guessing this file is no longer being used since it is called oldchat.html. Can it be deleted also?

Reply:
yup! but there are some issues right now that i am still using this as a reference for but this will be deleted soon.

Reply:
Can this be deleted now?